webhook_to_fedora_messaging/cli.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import os
+import sys
+from asyncio import run
+
+import click
+
+from webhook_to_fedora_messaging import __version__
+from webhook_to_fedora_messaging.config import setup_config, setup_database_manager
+from webhook_to_fedora_messaging.database import setup_database
+from webhook_to_fedora_messaging.main import start_service
+
+
+logger = logging.getLogger(__name__)
+
+
+@click.group()
+@click.option(
+    "-c",
+    "--conf",
+    "conf",
+    type=click.Path(exists=True),
+    help="Read configuration from the specified module",
+    default=None,
+)
+@click.version_option(version=__version__, prog_name="w2fm")
+def main(conf=None):
+    if conf:
+        os.environ["W2FM_CONFIG"] = os.path.abspath(conf)
+    try:
+        setup_config()
+        setup_database_manager()
```

Comment:
I think I would put that line outside of the try/except block to make it clear that we only expect `FileNotFoundError` to come from `setup_config()` (it's minor)